### PR TITLE
Integrate polarion TC CEPH-10650 with cephci

### PR DESCRIPTION
**Pull request by anrao19**
# Description
https://github.com/red-hat-storage/ceph-qe-scripts/pull/476
test case id: CEPH-10650

log: 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-lct5m/
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-lct5m/test_swift_enable_version_on_conatainer_of_different_user_0.log
http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/test_swift_enable_version_with_different_user.console.log

jira ticket :https://issues.redhat.com/browse/RHCEPHQE-9461

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by ckulal — COMMENTED** [review]
Looks  good, can be merged once ceph-ge-script Pr merged

**Review by TejasC88 — APPROVED** [review]

**Pull request merged**
